test/blackbox-tests/test-cases/melange/public.t/run.t

```diff
@@ -15,13 +15,13 @@ Cmj rules should include --bs-package-name
   $ output=my_project/output
 
 Js rules should include --bs-module-type
-  $ dune rules $output/node_modules/pkg.app/my_project/app/b.js |
+  $ dune rules $output/node_modules/pkg.app/b.js |
   > grep -e "--bs-module-type" --after-context=1
       --bs-module-type
       commonjs
 
 Js rules should include --bs-package-name
-  $ dune rules $output/node_modules/pkg.app/my_project/app/b.js |
+  $ dune rules $output/node_modules/pkg.app/b.js |
```

Comment:
Not related to this pr, but shouldn't this be `pkg/app`?

Reply:
It looks correct to me. the library name is `pkg.app`, we use that name for the `node_modules` folder.